x-pack/plugins/remote_clusters/common/constants.ts

```diff
@@ -26,3 +26,19 @@ export const API_BASE_PATH = '/api/remote_clusters';
 
 export const SNIFF_MODE = 'sniff';
 export const PROXY_MODE = 'proxy';
+
+export const getSecurityModel = (type: string) => {
+  if (type === 'certificate') {
+    return i18n.translate('xpack.remoteClusters.securityModelCert', {
+      defaultMessage: 'Certificate',
+    });
+  }
+
+  if (type === 'api_key') {
+    return i18n.translate('xpack.remoteClusters.securityModelApiKey', {
+      defaultMessage: 'API keys',
```

Comment:
```suggestion
      defaultMessage: 'API key',
```
Shouldn't it be singular like Certificate?